src/test/credentials/provider/ssoAccessTokenProvider.test.ts

```diff
@@ -131,11 +137,12 @@ describe('SsoAccessTokenProvider', () => {
             const stubCreateToken = sandbox.stub(ssoOidcClient, 'createToken')
             stubCreateToken.onFirstCall().returns(({
                 promise: sandbox.stub().throws({ code: 'AuthorizationPendingException' }),
-            } as any) as AWS.Request<CreateTokenResponse, AWS.AWSError>)
+            } as any) as SDK.Request<CreateTokenResponse, SDK.AWSError>)
+            clock.tickAsync('1')
```

Comment:
Why is this a string instead of an int? Ditto elsewhere.

Reply:
It can be either milliseconds or a readable string (ex. 2:30 for 2 minutes 30 seconds).  In this case I will change it to milliseconds to stay consistent with the rest of the file.